lib/api.db.mysql.js

```diff
@@ -140,10 +140,10 @@ if (api.mysql) {
               }
             }
             fieldDef = fieldType.type;
-            if ('size' in field) fieldDef += '(' + field.size + ')';
+            if (field.size) fieldDef += '(' + field.size + ')';
             fieldDef += ' ';
-            if (('signed' in field) && !field.signed) fieldDef += 'unsigned ';
-            if ((('nullable' in field) && !field.nullable) || field.master) {
+            if (field.signed && !field.signed) fieldDef += 'unsigned ';
```

Comment:
The diff has the answer about the intention. Before the change the condition checked that the field exists and evaluates to true, so the equivalent change would be just `field.signed === false` (if it's guaranteed to be a boolean), or, if the exact data type is unknown, `field.signed !== undefined && !field.signed` or `field.hasOwnProperty('signed') && !field.signed`.